### PR TITLE
CORE-88 - Laravel 5.6

### PR DESCRIPTION
**Pull request by CalumTowers**

### CODE REVIEW

File: resources/views/visit-transfer/site/application/referees.blade.php [hunk 1]
```diff
@@ -34,37 +34,39 @@
                             {!! Form::open(["route" => ["visiting.application.referees.post", $application->public_id], "method" => "POST"]) !!}
 
                             <div class="col-md-6">
-                                {!! ControlGroup::generate(
-                                    Form::label("referee_cid","Referee CID"),
-                                    Form::text("referee_cid"),
-                                    Form::help("Please ensure this is correct.")
-                                )->withAttributes(["id" => "refereeCidHelp"]) !!}
+                                <div id="refereeCidHelp">
```
Comment (atoff) [inline resources/views/visit-transfer/site/application/referees.blade.php]: Where we are keeping the control group divs, we need the div to use `class="form-group"` to ensure proper styling in bootstrap

File: resources/views/visit-transfer/site/dashboard.blade.php [hunk 2]
```diff
@@ -65,16 +69,20 @@
...
+                                    <button class="btn btn-success">START PILOT APPLICATION</button>
+                                </a>
                             @elseif($currentVisitApplication && $currentVisitApplication->is_in_progress && $currentVisitApplication->is_pilot)
-                                    {!! Button::primary("CONTINUE APPLICATION")->asLinkTo(route("visiting.application.continue", [$currentVisitApplication->public_id])) !!}
+                                <a href="{{ route('visiting.application.continue', [$currentVisitApplicatioon->public_id]) }}">
+                                    <button class="btn btn-primary" href="">CONTINUE APPLICATION</button>
+                                </a>
                             @elseif($currentTransferApplication)
-                                {!! Button::danger("You currently have a transfer application open.")->disable() !!}
+                                <button class="btn btn-danger" disabled="disabled">You current have a transfer application open.</button>
```
Comment (atoff) [inline resources/views/visit-transfer/site/dashboard.blade.php]: Typo

File: resources/views/visit-transfer/site/dashboard.blade.php [hunk 3]
```diff
@@ -107,16 +115,20 @@
...
+                                <button class="btn btn-success" href="">START ATC APPLICATION</button>
+                            </a>
                         @elseif($currentTransferApplication)
-                            {!! Button::primary("CONTINUE APPLICATION")->asLinkTo(route("visiting.application.continue", [$currentTransferApplication->public_id])) !!}
+                            <a href="{{ route('visiting.application.continue', [$currentTransferApplication->public_id]) }}">
+                                <button class="btn btn-primary">CONTINUE APPLICATION</button>
+                            </a>
                         @elseif($currentVisitApplication)
-                            {!! Button::danger("You currently have a visit application open.")->disable() !!}
+                            <button class="btn btn-danger" disabled="disabled">You current have a visit application open.</button>
```
Comment (atoff) [inline resources/views/visit-transfer/site/dashboard.blade.php]: Typo

File: resources/views/visit-transfer/site/application/referees.blade.php (hunk 1 above)
Comment (CalumTowers) [reply resources/views/visit-transfer/site/application/referees.blade.php]: I tried that and it totally messed the styling up. It looks fine in browser without - can you test a few of them please?

Comment (atoff) [reply resources/views/visit-transfer/site/application/referees.blade.php]: Fair enough, they look fine without them (although I don't really see a difference with them), so I would just suggest we leave them out for now.

**Pull request merged**